frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -127,6 +133,63 @@ const HostSummary = ({
     </div>
   );
 
+  const renderMacSettingsIndicator = () => {
```

Comment:
This might be a good candidate for its own React component.

Reply:
For sure. I have a [MacSettingsIndicator](https://github.com/fleetdm/fleet/pull/9809/files#diff-e23079f72b13bd34eb978eded467265dad4f366a6fece60cd52c887f355f92d1) component for the DataTable cells, which is pretty much the same format as this and I'll use here too.

Could also be combined with the "StatusIndicator" component that's used elsewhere, but that's blowing this up a bit much, perhaps.